frontend/src/Pages/SignUp.jsx

```diff
@@ -1,35 +1,102 @@
-import React, { useReducer } from "react";
+import React, { useReducer, useEffect, useState } from "react";
 import Button from "../UI/Button";
 import Card from "../UI/Card";
-import classes from "./SignUp.Module.css";
+import classes from "./SignUp.module.css";
```

Comment:
Wenn ich versuche deinen Branch zu runnen, bekomme ich 2 Errors, weil das File hier nicht gleich benannt ist, wie das eigentliche File `SignUp.Module.css`. einmal ist das M groß und einmal klein. Das muss einheitlich sein, da das File sonst nicht gefunden wird. Das gleiche mit `Card.Module.css`.

Reply:
Ich weiß. Bekomme das aber nicht (mehr) geändert. In meinem Branch sind die Dateien mit *.module.css benannt aber Git ist nicht case sensitive (glaube ich, vielleicht liegt's auch an meinen File System). Deshalb kommt die Änderung nicht ins repo. Hast du einen Hinweis für mich wie ich das hinbekomme?

Reply:
Ah, hab schon gefunden. Probiere das heute Abend mal aus. 

https://stackoverflow.com/questions/10523849/changing-capitalization-of-filenames-in-git